doc/nrf/ug_bootloader_fragments.rst

```diff
@@ -0,0 +1,29 @@
+.. _ug_bootloader_fragments:
+
+Assigning Kconfig fragments to bootloaders
+##########################################
+
+You can use Kconfig fragments specific to bootloaders in combination with project configurations using :ref:`ug_multi_image_variables`.
```

Comment:
There is one thing that I find very useful and I think it is missing in this guide.
When I create an application I want to be in control of the configuration so for me it is no-go for adding fragments on top of some configuration options that can possibly change in time without me knowing about it.

For this reason I think it should be shown that there is `b0_CONF_FILE` option. And that it can not only be passed to west (or cmake) but it can be set inside main cmake file of the application being created, so integrator can point to the full configuration file used by the project.

Reply:
I don't know if it should be added here or in some other chapter.

Reply:
Good point. I changed up this file to instead be a more general `ug_bootloader_config.rst` that contains a section focused on project configurations for more permanent configurations. The fragment section is then an addendum for more temporary configs.